tests/test_ordinal_df.py

```diff
@@ -0,0 +1,240 @@
+import jax.numpy as jnp
+import polars as pl
+import pytest
+
+from lpm_fidelity.counting import OrdinalDF
+
+
+def test_ordinal_df_single_column_two_values():
+    df = pl.DataFrame({"col1": ["a", "b", "a", "b"]})
+    odf = OrdinalDF.from_dataframe(df)
+
+    assert odf.columns == ("col1",)
+    assert len(odf.encoders) == 1
+    assert odf.data.shape == (4, 1)
+    assert set(odf.data[:, 0].tolist()) == {0, 1}
```

Comment:
There are plenty of assert testing shapes and unique values via `set`, but not a single one test against exactly the coding (I guess `[0, 1, 0, 1]` in this case). We should have that, no?